.github/workflows/release-internal.yml

```diff
@@ -32,20 +32,45 @@ on:
         required: true
         default: false
 
+  schedule:
+    - cron: '0 13 * * 3' # Every Wednesday at 8:00 AM central time
+
 defaults:
   run:
     shell: bash
 
 jobs:
+  get-latest-release:
+    runs-on: ubuntu-latest
+    outputs:
+      latest_release: ${{ steps.get_latest_release.outputs.latest_release }}
+    steps:
+      - name: Get latest release
+        id: get_latest_release
+        run: |
+          latest_release=$(curl --silent "https://api.github.com/repos/${{ github.repository }}/releases/latest" | jq -r .tag_name)
+          echo "::set-output name=latest_release::$latest_release"
+
+  check-for-new-commits:
+    runs-on: ubuntu-latest
+    needs: get-latest-release
+    outputs:
+      new_commits: ${{ steps.check_new_commits.outputs.new_commits }}
+    steps:
+      - name: Check for new commits since last release
+        id: check_new_commits
+        run: |
+          last_release_date=$(curl --silent "https://api.github.com/repos/${{ github.repository }}/releases/tags/${{ needs.get-latest-release.outputs.latest_release }}" | jq -r .published_at)
+          new_commits=$(git rev-list --count --since="$last_release_date" ${{ inputs.ref }})
+          echo "::set-output name=new_commits::$new_commits"
+
```

Comment:
do we need to point somewhere else for this? won't internal releases be far ahead of the version releases? can we look at previous runs of this workflow and grab the commit from them?

Reply:
have updated workflow from this initial comment would love thoughts.